pyproject.toml

```diff
@@ -5,10 +5,10 @@ requires = [
     "wheel",
     "Cython>=0.28.5",
 
-    # PyPy needs numpy >= 1.14.0
+    # PyPy needs numpy >= 1.19.0
     # platform_python_implementation!='CPython' not needed >= Python 3.7 which is numpy 1.14.5
     "numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation=='CPython'",
-    "numpy==1.14.0; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation!='CPython'",
+    "numpy==1.19.0; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation!='CPython'",
```

Comment:
Shouldn't we make it more explicit that this is for PyPy  with `platform_python_implementation=='PyPy'"` instead?

Reply:
We can probably remove `platform_machine!='aarch64' and platform_system!='AIX'` for PyPy as I don't think those platforms are supported by PyPy anyway.

Reply:
I believe both of these will be resolved with https://github.com/scikit-learn/scikit-learn/pull/18900, and that PR looks like it's going to completely rely on `oldest-supported-numpy`.